python/tidy/servo_tidy/tidy.py

```diff
@@ -104,6 +76,23 @@
     " accessible to\n// web pages."
 ]
 
+# Default configs
+config = {
```

Comment:
Why do you have a default config dict, but no default for `exclude` or `ignored_dirs`?